modules/custom/d_content/templates/page--node.html.twig

```diff
@@ -0,0 +1,63 @@
+{#
+/**
+ * @file
+ * Theme override to display a contente page .
+ *
+ * @see template_preprocess_page()
+ */
+#}
+{% set classes = [
+  'navbar-wrapper',
+  header_layout == 'header_with_cta' ? 'mb-lg-2',
+] %}
+{% extends "page.html.twig" %}
+{% block head %}
+  {% if header_layout != 'header_disabled' %}
+    <nav{{ navbar_attributes.addClass(classes) }}>
+      <div class="container no-padding-md">
+        <div class="navbar-brand d-flex align-items-center">
+          {{ page.header }}
+        </div>
+        {% if (header_layout == 'header_with_cta') %}
+          {% if node.field_header_cta and (node.field_header_cta.0.uri is not empty) %}
```

Comment:
Może powinniśmy tu po prostu wyrenderować pole z takim zastrzeżeniem, że w mobilu jego tekst zmieni się w ikonkę? Domyślnie w CTA link attributes możemy dodać `btn btn-primary`. Tę zmianę tekstu w ikonkę można ogarnąć przez podmianę jakiegoś template dla pola albo linku. Albo nawet przez JS.

Reply:
popieram, to ponizej wygląda na ciut zbyt skomplikowane

Reply:
może po prostu w `page.html.twig` który extendujemy, dodać `{% block cool-new-block %}`

![image](https://user-images.githubusercontent.com/37533838/86593174-0ac5f800-bf95-11ea-9097-8b7102b7adaa.png)

 i tutaj zrobić coś w stylu:

if header_disabled
{% block head %} {% endblock %}

if header_with_cta
{% block cool-new-block %} {{ pole z CTA }} {% endblock %}

unikniemy wielu powtórzeń chyba

Reply:
Pole CTA znajduje się w zmiennej node i nie znalazłem sposobu żeby go wyrenderować w tym miejscu inaczej. Dodałem domyślne klasy dla odnośnika (btn i btn-primary) a całość kodu do szablonu page.html.twig